CONTRIBUTING.md

```diff
@@ -8,9 +8,12 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 > Install yarn on your system: https://yarnpkg.com/en/docs/install
 
 ```sh
-$ git clone https://github.com/yarnpkg/yarn.git
+$ # fork the repo
+$ git clone YOUR_YARN_REPO_URL
```

Comment:
YOUR_YARN_REPO_URL is more confusing, it implies that a person forked yarn on github